theorems/homotopy/FinSet.agda

```diff
@@ -0,0 +1,97 @@
+{-# OPTIONS --without-K --rewriting #-}
+
+open import HoTT
+import homotopy.ConstantToSetExtendsToProp as ConstExt
+open import homotopy.Pigeonhole
+
+module homotopy.FinSet where
+
+-- the explicit type of finite sets, carrying the cardinality on its sleeve
+FinSet-exp : Type₁
+FinSet-exp = Σ ℕ λ n → BAut (Fin n)
```

Comment:
Is there a reason to restrict the construction to the first universe? This seems to work in arbitrary finite universes.

Reply:
I see why now.